### PR TITLE
chore: update to .NET 8.0.101

### PR DESCRIPTION
**Pull request by vaind**
#skip-changelog

### CODE REVIEW

File: global.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "sdk": {
-    "version": "8.0.100",
+    "version": "8.0.101",
     "rollForward": "latestMinor",
```
Comment (bruno-garcia) [inline global.json:4]: it should roll forward already, do we need this?

Comment (vaind) [reply global.json:4]: hmm, guess not then

Comment (SimonCropp) [reply global.json:4]: it only rolls forward if people have it installed. 

I would think you would prefer not to build from an sdk with a CVE?

Comment (bruno-garcia) [reply global.json:4]: By reading the CVE it doesn't seem to affect building a library. It's a runtime thing

That said I don't mind pinning it to a newer version. I just reviewed it as-is without the context on why (usually we don't bump for patch releases on global.json so any version from the base is OK to make things work)

**Pull request closed without merging**